docker-compose.yml

```diff
@@ -63,7 +66,11 @@ services:
       - fhir-api
     environment:
       SERVER_PORT: "8083"
-      SPRING_PROFILES_ACTIVE: "local"
+      SPRING_PROFILES_ACTIVE: ${PROFILE:-local}
+      FHIR_URI: "http://fhir-api:8080"
```

Comment:
setting this env var of fhir_uri takes precedence over the values set in the config file, meaning you couldn't ever change this when running it with compose. is it wise/necessary?
(Applies for the other services in this file)

Reply:
it is needed to keep both the fhir uri here and in the '-local.yml' file.
if we run the services locally without docker compose, the yml file is needed to set the value.
if running the docker compose then you can set ANY profile and you will get the correct fhir uri. so also needed